scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -72,7 +72,7 @@
     </div>
     {% endif %}
 
-    <div class="row"><div class="col-12">{{ matching_variants(managed_variant, causatives) }}</div></div>
+    <div class="row"><div class="col-12">{{ matching_variants(managed_variant, causatives, variant.matching_ranked) }}</div></div>
```

Comment:
This works but the alignment is off

<img width="341" alt="image" src="https://user-images.githubusercontent.com/28093618/212840488-30f25f17-c836-4a68-ae2e-92964b37930f.png">

Reply:
Ah! Same as the Tiered, but looks a little odd once you see it. I'll see what can be done.

Reply:
Like so:
![Screenshot 2023-01-31 at 11 15 46](https://user-images.githubusercontent.com/758570/215733102-e7de0938-19d9-457c-b741-eea422d39e4d.png)